app/controllers/users.js

```diff
@@ -0,0 +1,32 @@
+const { logger } = require('express-wolox-logger');
+const { errors, statusCode } = require('../helpers');
+const { userMapper } = require('../mappers');
+const { UserService } = require('../services');
+
+const signUp = async (req, res, next) => {
+  try {
+    const userDTO = userMapper.signUpMapper(req.body);
```

Comment:
```suggestion
    const userDTO = userMapper.signUpDTO(req.body);
```
I think it is more meaningful if you avoid writing mapper more than once